### PR TITLE
Integration with BrightGauge

### PR DESCRIPTION
**Pull request by Madelene**

### CODE REVIEW

File: integrations.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+
+-
+  name: "BrightGauge Software"
+  description: "Track data about Basecamp usage through dashboards, gauges and reports, alongside other integrations, including Harvest and Dropbox."
+  website: "https://www.brightgauge.com/"
+  icon: "brightgauge.png"
+  category: 3
```
Comment (rosa) [inline integrations.yml]: Thanks @Madelene! Would you mind moving this to the end of the file instead of having it here at the top?

**Comment by rosa** [comment]
Thanks @Madelene! I've been looking at https://www.brightgauge.com/, and [the integrations page](https://www.brightgauge.com/integrations), but can't find anything specific about Basecamp. Could you point me to the right place?

**Comment by Madelene** [comment]
We just went live with the integration this week. Marketing should get the article out sometime next week! Our Director of Marketing would love to talk to someone, on your end, about co-marketing opportunities. Is it possible to put him in contact with someone regarding this?

**Comment by rosa** [comment]
Perfect, thanks @Madelene! I prefer to wait until you've published the details on the integration to merge this. If possible, when that happen, could you update the website URL to be the specific place where you describe the Basecamp integration instead of linking directly to `"https://www.brightgauge.com/"`? 

When you do that I'll get this merged and added to `/extras` right away. 

> Our Director of Marketing would love to talk to someone, on your end, about co-marketing opportunities. Is it possible to put him in contact with someone regarding this?

Sorry! We only add 3rd party integrations to `/extras` but don't do any co-marketing. Thanks for offering!

**Comment by Madelene** [comment]
Sounds great! I'll update this sometime next week (marketing says it will be published next week) 😸

**Comment by Madelene** [comment]
Heya, @rosa ! The URL is updated with the exact link. 😸

**Pull request merged**